vars/documentationChecker.groovy

```diff
@@ -0,0 +1,51 @@
+
+
+// Add the following code to any Jenkinsfile.
+// It will return with success if there are no changes in the specified docs dir
+// and Will return with success if run on a non Pull Request branch
+
+// documentationChecker("docs")
+
+// docsDir ~~ the subdirectory within your repo that the documentation resides
+def documentationChecker(docsDir) {
+  if (env.CHANGE_ID != null) {
```

Comment:
ah you know what, i was getting `BRANCH_NAME` and `CHANGE_ID` confused. `BRANCH_NAME` is what has 'PR-123' on PRs and branch name for branches (as one would hope). appears you're right that `CHANGE_ID` is only set on PRs.